src/hookable.ts

```diff
@@ -71,6 +71,14 @@ export class Hookable<
       }
     }
 
+    // Add name to hook for better debugging experience
+    if (!function_.name) {
+      Object.defineProperty(function_, "name", {
+        get: () => name + " callback",
```

Comment:
since it is a function name, and we don't want to introduce a dep, maybe replace `:` to `_` and prefix with hook_cb`? (Example: `nuxt:app:test` => `nuxt_app_test_hook_cb`)